modules/ossm-jaeger-config-elasticsearch.adoc

```diff
@@ -0,0 +1,43 @@
+////
+This TASK module included in the following assemblies:
+-ossm-configuring-jaeger.adoc
+////
+
+[id="ossm-jaeger-config-elasticsearch_{context}"]
+= Configuring the Elasticsearch index cleaner job
+
+When the {ProductShortName} Operator creates the `ServiceMeshControlPlane` it also creates the custom resource definition (CRD) for Jaeger.  The Jaeger operator then uses this CRD when creating Jaeger instances.
```

Comment:
The service mesh creates a "custom resource (CR) for Jaeger" - not CRD. "uses this "CR" when creating the Jaeger instance"